code/mathematical-algorithms/factorial/factorial.cs

```diff
@@ -0,0 +1,23 @@
+using System;
```

Comment:
Kindly remove this code as the recursive implementation in `C#` exists 👍

Reply:
done kindly , review